styles.css

```diff
@@ -0,0 +1,946 @@
+    font-weight: 400;
+    padding: 0 202px;
+  }
+
+  .contactMe .form {
+    width: 327px;
+    margin: auto;
+  }
+
+  .contactMe .form-control {
+    margin: 10px auto;
+  }
+
+  .contactMe input {
+    border-radius: 8px;
+    width: 100%;
+    height: 48px;
+    margin: 14px 0;
+    padding: 10px;
+  }
+
+  .contactMe textarea {
+    border-radius: 8px;
+    width: 100%;
+    height: 170px;
+    margin: 14px 0;
+    padding: 10px;
+  }
+
+  .contactMe .buttonThree {
+    display: block;
+    flex-direction: row;
+    align-items: center;
+    padding: 12px 16px;
+    background: #fff;
+    border-radius: 8px;
+    color: #396df2;
+    font-weight: 500;
+    font-size: 17px;
+    line-height: 24px;
+    margin: auto;
+  }
+
+  .buttonThree:hover {
+    color: white;
+    background-color: #396df2;
+  }
+}
+
+/* mobile version */
+
+@media (max-width: 767.9px) {
+  body {
+    font-family: "Poppins", sans-serif;
+    background-color: #e5e5e5;
+    width: 100%;
+  }
+
+  #navbar {
+    padding: 20px;
+    display: flex;
+    justify-content: space-between;
+    top: 0;
+    overflow: hidden;
+    position: fixed;
+    height: 60px;
+    background-color: #fff;
+    width: 100%;
+  }
+
+  .langu ul {
+    display: flex;
+    flex-direction: column;
+    list-style-type: none;
+  }
+
+  .new {
+    position: fixed;
+    left: -100%;
+    top: 60px;
+    gap: 0;
+    flex-direction: column;
+    background-color: #fff;
+    width: 100%;
+    text-align: left;
+  }
+
+  .new.active {
+    left: 0;
+    height: 100%;
+    background-color: #6070ff;
+    top: 0;
+    opacity: 0.95;
+  }
+
+  .nav-menu {
+    margin-top: 70px;
+  }
+
+  .humburger {
+    cursor: pointer;
+    display: block;
+    color: #6070ff;
+  }
+
+  .bar {
+    display: block;
+    width: 25px;
+    height: 3px;
+    margin: 5px auto;
+    background-color: #6070ff;
+  }
+
+  .humburger.active .bar:nth-child(2) {
+    opacity: 0;
+  }
+
+  .humburger.active .bar:nth-child(1) {
+    transform: translateY(8px) rotate(45deg);
+    background-color: #fff;
+  }
+
+  .humburger.active .bar:nth-child(3) {
+    transform: translateY(-8px) rotate(-45deg);
+    background-color: #fff;
+  }
+
+  a:hover {
+    color: cyan;
+  }
+
+  #navbar ul a {
+    padding: 4px;
+    margin: 0 10px;
+    color: #fff;
+    font-size: 32px;
+    font-weight: 600;
+    text-decoration: none;
+  }
+
+  .lang li {
+    background-color: #ebebff;
+    border-radius: 10px;
+    color: #6070ff;
+    padding: 4px 12px;
+    margin-bottom: 6%;
+    font-size: 12px;
+    font-weight: 500;
+  }
+
+  .listItem li {
+    margin-right: 5px;
+    font-size: 18px;
+    font-weight: 400;
+    color: #6b778c;
+    margin-left: 20px;
+  }
+
+  .listItem li:first-child {
+    font-weight: 500;
+    color: #344563;
+    list-style-type: none;
+    margin-left: 0;
+  }
+
+  .languag .fLi li {
+    padding: 5px;
+    width: 25%;
+  }
+
+  #navbar ul li {
+    list-style-type: none;
+    margin-left: 20px;
+  }
```

Comment:
[OPTIONAL]
- [ ] Kindly notice the margin between each links in the mobile menu. Please add `margin-bottom: 20px` to follow the Figma design as requested in the project requirement.
### Your design :point_down: 
![Screenshot from 2023-05-15 17-19-10](https://github.com/abdozayan12/portfolio-include-js/assets/91142432/afb281f3-c1f0-42aa-9cfc-3b34da8eeb89)

### Figma design :point_down: 
![Screenshot from 2023-05-15 17-26-08](https://github.com/abdozayan12/portfolio-include-js/assets/91142432/e72fdfe6-1c7f-47c8-9081-d961b712b9d8)